src/components/common/CommonButtons/CommonButtons.jsx

```diff
@@ -0,0 +1,18 @@
+import { S } from './CommonButtons.style';
+
+export const TextIconButton = ({ text, icon }) => {
+  return (
+    <S.JoinUsContainer>
```

Comment:
```suggestion
export const TextIconButton = ({ text, icon, bgColor, border }) => {
  return (
    <S.JoinUsContainer $bgColor={bgColor} $border={border}>
```
이 버튼으로 같이 쓸 수 있는데 색이랑 border만 다른 게 있어서 추가하면 좋을 것 같아요!